examples/ref-implementation/README.md

```diff
@@ -0,0 +1,121 @@
+# Reference implementation 
+
+This example creates a local version of the CNOE reference implementation.
+
+## Installation
```

Comment:
Can you also add a prerequisite section as I suspect that we need:
- kubectl installed + kube OIDC login plugin (?),
- idpbuilder version 0.x (the x should be defined ;-))
- An account on Amazon and S3 bucket,